src/components/legend/draw.js

```diff
@@ -514,6 +548,9 @@ function computeLegendDimensions(gd, groups, traces) {
     var fullLayout = gd._fullLayout;
     var opts = fullLayout.legend;
     var gs = fullLayout._size;
+    opts._titleWidth = fullLayout._legendTitleWidth;
+    opts._titleHeight = fullLayout._legendTitleHeight;
```

Comment:
Why do we need to stash `_legendTitleWidth` and `_legendTitleHeight` in both `fullLayout` AND `fullLayout.legend`?

Reply:
Good point. 
Fixed in https://github.com/plotly/plotly.js/pull/4386/commits/a9759a27029ce39dd1071d570b8254d0ea667d00.